test-cases/tests/authorization/b09-validate-authentication-with-ldap-idp.md

```diff
@@ -0,0 +1,117 @@
+---
+# See the metatadata section in the README.md for details on the
+# allowed fields and values
+automation:
+  - INTLY-0000
```

Comment:
should this be updated?

Reply:
I'll remove field as we don't have any ticket to automate this test case